src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/ReportVersionViewModel.kt

```diff
@@ -44,20 +46,23 @@ data class ReportVersionPageViewModel(@Serialise("reportJson") val report: Repor
                     focalArtefactUrl,
                     isAdmin,
                     isRunner,
+                    isUsersManager,
                     artefacts,
                     dataLinks,
                     resources,
                     zipFile,
                     versions,
                     changelog,
+                    reportReaders,
                     DefaultViewModel(loggedIn, appName, breadcrumbs))
 
     companion object
     {
         fun build(report: ReportVersionDetails,
                   versions: List<String>,
                   changelog: List<Changelog>,
-                  context: ActionContext): ReportVersionPageViewModel
+                  context: ActionContext,
+                  authRepo: AuthorizationRepository): ReportVersionPageViewModel
```

Comment:
So this changes the pattern a bit, as so far we've not passed repos into the builder, but retrieved data from repos in the controller first and then passed the data along to the builder. I don't have a strong feeling about which pattern is better, maybe we should be passing repos here, but I do think we should be consistent.

Reply:
Yeah, I was aware of that, but the reason I wanted to do it this way was just so that we didn't bother retrieving report readers if the user doesn't have users.manage permissions, and we've got the pattern that the builder determines the permissions.
 
It had struck me that the is.. properties were getting a bit unwieldy so maybe one approach would be to make UsersReportPermissions a separate model (with isAdmin etc on it), get the controller to have that build itself, then the controller decides whether to fetch report readers to pass to the ReportVersionViewModel builder based on its isUsersManager value. What do you think?

Reply:
Oh! And actually, this goes away in the branch I'm working on at the moment anyway (for adding readers from the page) since it seemed to be simpler to just have an endpoint for fetching report readers, and have the vue component (which is only there if isUsersManager) refresh that on mount and whenever the list changes - so acutally I've already refactored this away...